tradehub/authenticated_client.py

```diff
@@ -64,7 +64,7 @@ def set_transaction_standards(self, messages: list, transaction_type: str, fee:
             fee_dict = fee
         else:
             fee_type = self.get_transaction_fee_type(transaction_type)
-            fee_amount = self.fees[fee_type] * len(messages)
+            fee_amount = str(int(self.fees[fee_type]) * len(messages))
```

Comment:
This one has me questioning what the correct way to handle this is, on line 139 there is a different handling of fees and I think we should probably just make a function that standardizes this so it can be called from anywhere.

Line 139 has the following

`fee_amount = to_tradehub_asset_amount(amount = len(messages), decimals = self.tokens["swth"]["decimals"])`

And this one doesn't handle the decimals at all. Also, this instance is going to be run pretty much everytime b/c I doubt people are going to be sending their own Fee instance. Almost has me wondering how this is working at all...

Reply:
Sounds better to handle this in one place. I think it worked for you, because you only tried single transactions. I tried multiple orders(CreateOrder ha 0 fees I know just want to show you why it worked). 
Your case: "1000000" * 1 = "1000000"
My case: "1000000" * 3 = "100000010000001000000"
Worked because is it was correct.

Reply:
I was really sloppy here, thanks for the explanation!

Since the fees endpoint returns a string it messes up the multiplication and you are correct, I hadn't tested multiple orders, only single orders.

And in the case of line 139, it is a rudimentary attempt at getting fees and is even more incorrect than this.

I think we go ahead and accept this and then fix with a better solution in the next pull request.